### DISCUSSION TITLE
Visualizing multiple layers

### DISCUSSION THREAD

**Discussion by giswqs** [developmentseed/lonboard#143] [Q&A] [answered]
Excited to see this new package. Does it support visualizing multiple layers?  It seems both `viz` and the `Layer` class returns a map. Not sure it if feasbile to stack multiple layers on the map.

**Comment by kylebarron** [comment, marked as answer]
It doesn't yet; we need to figure out https://github.com/developmentseed/lonboard/pull/34 and https://github.com/developmentseed/lonboard/issues/33 to support multiple layers on a single map

**Comment by kylebarron** [comment]
This is supported in 0.2: https://github.com/developmentseed/lonboard/releases/tag/v0.2.0
